src/components/service-metrics/controllers.tsx

```diff
@@ -35,6 +35,8 @@ import {
   UnsupportedServiceMetricsPage,
 } from './views';
 
+const PERSISTANCE_MESSAGE_15_DAYS = '15 days';
```

Comment:
We persist metrics for up to `370d`

https://github.com/alphagov/paas-cf/blob/master/manifests/cf-manifest/operations.d/500-prometheus.yml#L57